news/2022-08-01-osumonthly-1.md

```diff
@@ -0,0 +1,224 @@
+
+![](/wiki/shared/news/2022-08-01-osumonthly-1/roundtable/RTable_players.jpg)
+
+With the players showcased above, we have [BTMC](https://osu.ppy.sh/users/3171691) and [Happystick](https://osu.ppy.sh/users/256802) commentating, genius roster who make sense for this first **IRL osu! invitational**!
+
+As if it wasn't enough, [mrekk](https://osu.ppy.sh/users/7562902) was there too! If you weren't able to follow the event, no worries! The official Roundtable's YouTube channel has the full VODs [here](https://www.youtube.com/channel/UC6fSQd4oKBPxkBR93GucCUg/featured)!
+
+With that many great players, such as [mrekk](https://osu.ppy.sh/users/7562902), [Utami](https://osu.ppy.sh/users/7512553), [RyuK](https://osu.ppy.sh/users/6304246) and many others, this tournament was incredible! The competition's spirit was mixed with the entertaining cast and gimmicks of [BTMC](https://osu.ppy.sh/users/3171691) and the result was epic!
+
+In terms of epic, we were served with the finals: [mrekk](https://osu.ppy.sh/users/7562902) vs [Utami](https://osu.ppy.sh/users/7512553)! Despite mrekk's defeat against Utami, his performances were impressive, along with Utami's. His first rank on the leaderboard is indeed his throne.
+
+A best-of from this match was made on the BTMC channel, with exclusive interviews of the players, by BTMC (what a surprise, ahah) and Happystick, the casters of this amazing show! You can find it [here](https://www.youtube.com/watch?v=Y7LqBZL)!
+
+Anyway, The Roundtable has now ended, and hopefully this towering experience returns next year!
+
+[*Find the forum article here!*](https://osu.ppy.sh/home/news/2022-07-08-the-roundtable)
+
+Thanks again to [BTMC](https://osu.ppy.sh/users/3171691) and his staff for this insane event! 
+
+### ![](/wiki/shared/mode/taiko.png) osu!taiko
+
+As if we could allow for osu! to take all the glory in its showcasing of top players and their performances, osu!taiko takes it upon itself to rival even that of *The Roundtable* with their very own annual entry of **[Expert Global Taiko Showdown 2022](https://osu.ppy.sh/wiki/en/Tournaments/GTS/EGTS_2022)**!
+
+As the name implies, the showdown is to take place between those whose dreams consist of taiko drumming, kiai slider velocity shenanigans, dons, katsus, and if we even dare-say, taikonator 3000! This is a tournament that showcases those whose passion for Taiko is not just exemplary in their gameplay but in every other way. 
+
+### ![](/wiki/shared/mode/catch.png) osu!catch
+
+*At the beginning of July, the recent eleventh iteration of osu!catch World Cup has been concluded.*
+
+For the fourth time in a row and a total of eight wins, South Korea are the osu!catch champions. The roster consisting of JISAN, Rells, YesMyDarknesss, x_angelkawaii_x, DreStar and Byeol. With second and third places placed by United States (Secre, Lexii, Dahcreeper, Zak, Colin, Elux) and China (Dusk, Callionet, Coevir, Crystal, KagamineRin02, Kitakami).
```

Comment:
For the fourth time in a row and a total of eight wins, South Korea are the osu!catch champions. The roster consists of [JISAN](https://osu.ppy.sh/users/3657951), [Rells](https://osu.ppy.sh/users/7937519), [YesMyDarknesss](https://osu.ppy.sh/users/4158549), [x_angelkawaii_x](https://osu.ppy.sh/users/566276), [DreStar](https://osu.ppy.sh/users/1808057) and [Byeol](https://osu.ppy.sh/users/2754651). With second and third places placed by United States ([Secre](https://osu.ppy.sh/users/2306637), [Lexii](https://osu.ppy.sh/users/7226149), [Dahcreeper](https://osu.ppy.sh/users/6926006), [Zak](https://osu.ppy.sh/users/1375955), [Colin](https://osu.ppy.sh/users/5502521), [Elux](https://osu.ppy.sh/users/12004983)) and China ([Dusk](https://osu.ppy.sh/users/533210), [Callionet](https://osu.ppy.sh/users/3072921), [Coevir](https://osu.ppy.sh/users/3171965), [Crystal](https://osu.ppy.sh/users/1646397), [KagamineRin02](https://osu.ppy.sh/users/518503), [Kitakami](https://osu.ppy.sh/users/6347308)).